packages/eslint-plugin-pf-codemods/lib/helpers.js

```diff
@@ -552,8 +552,9 @@ function addCallbackParam(componentsArray, propMap) {
                       const createRemoveCurrentParamUseFix = (
                         currentUseOfNewParam
                       ) => {
+                        let tokenBeforeCurrentUse = context.getTokenBefore(currentUseOfNewParam)
```

Comment:
Nit: can we make this a const instead of let?

Reply:
Yep, you're right, thanks.